### ISSUE TITLE
improve voice message accessibility

### ISSUE THREAD

**Issue by cyBerta** [deltachat/deltachat-ios#1040]
A user reported back the following issues with voice messages 

> * Der Schalter zum Wiedergeben der Sprachnachricht ist nicht beschriftet.
> * Man weiß nicht, von wem die Sprachnachricht ist, also in dem Verlauf 
> selber.
> * Man sieht nicht, ob eine Sprachnachricht noch ungelesen ist. Ggf. gibt 
> es auch noch andere Flags.
> 

so we should:
- add an accessibility label to the play button and ensure starting and stopping of voice messages work 
-  add the sender name  that the user can choice beforehand to skip a message

- wrt. the unread message state we might wait until we introduced the unread message divider in a chat on iOS. After implementing that, the first visible message at the top should be the first unread message. Voice over should then just start with that first unread message (maybe even with the unread message label to inform the user that the follwing messages haven't been read yet).
